### PR TITLE
Fix: add from_height to improve btc sync

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description

Add a variable "syncedBlockHeight" to save the last synced block height. So that when we call
the 
https://explorers.api.live.ledger.com/blockchain/v4/btc/address/{address}/txs
endpoint to retrieve transactions. We can have higher block height.
This change can improve the btc sync performance.

### ❓ Context

- **JIRA or GitHub link**: 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** Hard to add automatic tests. because we just speed up the synchronization by changing https request parameters.
- [X] **Impact of the changes:** No impact. Only non-reg test is required
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.estimateMaxSpendable.integration.test.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe("testing estimateMaxSpendable", () => {
   });
 
   it("should estimate max spendable correctly", async () => {
-    await wallet.syncAccount(account);
+    await wallet.syncAccount(account, 1000000);
```
Comment (hedi-edelbloute) [inline libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.estimateMaxSpendable.integration.test.ts:29]: Minor but I think we should run a full sync in integration tests

**Review by hedi-edelbloute — DISMISSED** [review]
lgtm

add unit tests if possible :pray:

File: libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts [hunk 2]
```diff
@@ -141,14 +141,18 @@ class BitcoinLikeExplorer implements IExplorer {
   async getTxsSinceBlockheight(
     batchSize: number,
     address: Address,
-    startingBlockheight: number,
+    startBlockheight: number,
```
Comment (hedi-edelbloute) [inline libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts:144]: We could use "from" and "to" annotation to match backend parameters

File: libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.estimateMaxSpendable.integration.test.ts (hunk 1 above)
Comment (hzheng-ledger) [reply libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.estimateMaxSpendable.integration.test.ts]: fixed

**Comment by hzheng-ledger** [comment]
> lgtm
> 
> add unit tests if possible 🙏

It is hard to write unit tests.
As the main change in this PR is the "getTxsSinceBlockheight" method and "syncAccount" method, they depend on the backend response. These methods are covered indirectly by previous integration tests. (e.g. xpub.syncing.integration.test.ts)
We can just make sure that it doesn't break previous tests.

**Review by hedi-edelbloute — APPROVED** [review]

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**